TAScheduler/AcceptanceTests/AssignTAtoLab tests.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from django.test import TestCase
+
+from TAScheduler.models import User, TA, Course, TAToCourse
+
+
+#
+class SuccessfulAssignment(TestCase):
+    user = None
+    ta = None
+    course = None
+    tatocourse = None
+
+    def setUp(self):
+        self.user = User.objects.create(email_address="ta@class.com", password="password", first_name="first",
+                                        last_name="last", home_address="there", phone_number=1234567890)
+
+        self.ta = TA.objects.create(user=self.user)
+
+        self.course = Course.objects.create(course_id=100, semester="fall 2023", name="test",
+                                            description="test", num_of_sections=2, modality="online", credits=1)
+
+        self.tatocourse = TAToCourse.objects.create(ta=self.ta, course=self.course)
+
+    def test_creation(self):
```

Comment:
This test right here is testing if the things that we put int the database match what we put in the database. This test needs to be checking if the website can add a TA to course through a form. This was not done here